src/server/game/Entities/Creature/Creature.cpp

```diff
@@ -1688,6 +1688,10 @@ void Creature::DeleteFromDB()
     stmt->setUInt32(0, m_spawnId);
     trans->Append(stmt);
 
+    stmt = WorldDatabase.GetPreparedStatement(WORLD_DEL_CRELINKED_RESPAWN);
+    stmt->setUInt32(0, m_spawnId);
+    trans->Append(stmt);
```

Comment:
is this missing in gameobject too?

Reply:
Yes! I noticed it afterwards but had already done the pull request. Do you think I should add it here too?

Reply:
yes